### PR TITLE
Zulu Language is accpted but not show on binary transaltion page

### PR DESCRIPTION
**Pull request by rra0137**
link : https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/installer/nativeLang/zulu.xml

binary-translation : https://notepad-plus-plus.org/contribute/binary-translations.html

### CODE REVIEW

**Comment by rra0137** [comment]
@donho Would you please check Zulu language not show on binary-translations page

File: PowerEditor/installer/nativeLang/zulu.xml [hunk 1]
```diff
@@ -0,0 +1,1183 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/zulu.xml]: This file is already in master, it shouldn't show again.

**Comment by rra0137** [comment]
@don ho ..i have checked on binary page but i not found zulu binary translation that page

**Comment by donho** [comment]
zulu.xml is in master :
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/installer/nativeLang/zulu.xml
Obviously you work always on your old branch (master). update your branch and create a new branch to do your PR.

**Comment by donho** [comment]
it's already done this PR:
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/b1285e7b1e8e99aa3c89c93ccbe112bf9595bb6d

**Pull request closed without merging**